script.js

```diff
@@ -0,0 +1,65 @@
+const menubtn = document.querySelector('.humburger');
+const mobilemenu = document.querySelector('.mobile-nav');
+
+menubtn.addEventListener('click', () => {
+  menubtn.classList.toggle('is-active');
+  mobilemenu.classList.toggle('is-active');
+});
```

Comment:
As requested by the previous reviewer, please update your mobile menu functionality 😄. It is supposed to close when the user clicks on any of the `nav` links. Currently, it isn't closing, which is blocking the page view:

Opened mobile menu:
![image](https://user-images.githubusercontent.com/42799579/210602792-17206597-4fdc-4239-abe5-79f5b47d8fa8.png)

After clicking on 'Program' (here is what you should update):
![image](https://user-images.githubusercontent.com/42799579/210602934-cd1fe3cd-5ec4-4b1b-a375-a2d24a2b4f57.png)

Kindly notice that the page changed but the menu is still open! Make sure to close it when redirecting the user 😉.